Documentation/ProjectFile/properties/property/CapillaryPressureVanGenuchten/t_residual_gas_saturation.md

```diff
@@ -0,0 +1 @@
+The smallest degree of saturation of the gas (or non-wetting) phase in the medium.
```

Comment:
Need to discuss together with @Scinopode and @boyanmeng about the parameter. The class handles the function of p_c(S^l), one needs to restriction the variable of S^l in (S^l_r, S^l_max). Input S^g_r then convert it to S^l_max is weird, especially for single phase flow.

Reply:
This property is required only by multi-phase problems and is especially written to be used in the TwoPhaseFlowWithPrho process, isn't it?

Reply:
The p(S) is not only used for two-phase flow. The Richards' equation can also use the saturation as primary variable.

Reply:
By the way, this naming some how is related to the relative permeability, which should be a property of phase. For the relative permeability, these two restriction variables were suggested to remove phase info from their names (#2902) as
```
residual_saturation
maximum_saturation
```
after the relative permeability property is moved to phase property.